code/modules/shuttles/shuttle_specops.dm

```diff
@@ -45,9 +37,9 @@
 		C.visible_message("<span class='notice'>The Special Operations shuttle will depart in [(specops_countdown_time/10)] seconds.</span>")
 
 	if (location)	//returning
-		radio_announce("THE SPECIAL OPERATIONS SHUTTLE IS PREPARING TO RETURN")
+		do_telecomms_announcement(current_location, "THE SPECIAL OPERATIONS SHUTTLE IS PREPARING TO RETURN", "A.L.I.C.E")
```

Comment:
/datum/shuttle has a name/display_name var that you could use for announcements instead
`name = "A.L.I.C.E"` or `display_name = "A.L.I.C.E"`
`do_telecomms_announcement(current_location, "blah", name)`

Reply:
Oh true, I initially interpreted this as the shuttle announcing it's own departure.